### PR TITLE
feat: filterProperty supports multiple replacements

### LINKED ISSUE #17: option to append PR # at end of commit message for merge
need to support templates to accomplish this.

```
message: {title} (#{pull_request_number})
```

### PR DESCRIPTION
**Pull request by connorjclark**
fixes #17 

working here: https://github.com/connorjclark/git3po-test-repo/commit/d24ed1d10068b61559ed9f63b433d860d7250f46

### CODE REVIEW

File: lib/filter.js [hunk 1]
```diff
@@ -99,8 +100,8 @@ class Filter {
   }
 
   static replaceProperty(object, needle, replacement) {
-    if (object === needle) {
-      return replacement
+    if (typeof object === 'string') {
```
Comment (patrickhulce) [inline lib/filter.js:103]: can we make this an `else if`? I don't *think* we have any other cases, but it'd be nice to leave this working for numbers or whatever

File: lib/filter.js [hunk 2]
```diff
@@ -99,8 +100,8 @@ class Filter {
...
+      return object.replace(needle, replacement)
```
Comment (patrickhulce) [inline lib/filter.js:104]: feels like the line is blurring between `replaceProperty` and `replacePropertyDynamic` in ways I'm not going to understand later, but this works for me for now :)

File: lib/action.js [hunk 3]
```diff
@@ -45,9 +45,9 @@ class Action {
         action = preprocess(action, issue)
         return github.addAssignee(issue, action.login)
       case Action.TYPES.MERGE:
-        // By default, use the PR's title as the commit title.
+        // By default, use the PR's title + pr number as the commit title.
         if (!('title' in action)) {
-          action.title = '%%title%%'
+          action.title = '%%title%% (#%%pull_request_number%%)'
```
Comment (patrickhulce) [inline lib/action.js:50]: oh we were already using a template, bonus! :)

File: lib/filter.js (hunk 1 above)
Comment (connorjclark) [reply lib/filter.js:103]: We don't–I checked. Only called in one place with `%%${name}%%`

File: lib/filter.js (hunk 2 above)
Comment (connorjclark) [reply lib/filter.js:104]: haha yeah they are very similar. only difference now is one takes parameters and one doesnt. could be rolled into one with some minor effort ...

**Pull request merged**